tests/robot-tests/tests/libs/no_javascript.py

```diff
@@ -0,0 +1,15 @@
+from robot.libraries.BuiltIn import BuiltIn
+sl = BuiltIn().get_library_instance('SeleniumLibrary')
+import os
+import requests
+from bs4 import BeautifulSoup
+
+def user_checks_page_accordions_work_with_no_javascript(url):
+    response = requests.get(url)
+    soup = BeautifulSoup(response.text, "html.parser")
+    first_accordion = soup.select_one("[data-testid='accordionSection']")
+    if first_accordion.name != "div":
```

Comment:
What do you get if the selector can't find an element? Shouldn't it return a None or null-like value that we should be checking against instead?